modules/js/toggleSection.js

```diff
@@ -0,0 +1,4 @@
+export default function toggleSection(sectionId) {
+  const section = document.getElementById(sectionId);
+  section.style.display = section.style.display === 'none' ? 'block' : 'none';
+}
```

Comment:
- [ ] Consider using arrow functions instead of the `function` keyword to leverage ES6 features in JavaScript. Arrow functions offer concise syntax and prevent potential scoping bugs. Please give it a try to enhance code readability and maintainability.